templates/_devcontainer/nginx/nginx.conf.ts

```diff
@@ -20,15 +20,19 @@ http {
     listen 80;
 
     location / {
+      error_page 418 = @websocket;      
```

Comment:
remove trailing spaces:
```suggestion
      error_page 418 = @websocket;
```